service/src/main/java/org/apache/griffin/core/job/JobServiceImpl.java

```diff
@@ -128,10 +129,22 @@ Licensed to the Apache Software Foundation (ASF) under one
     @Autowired
     private StreamingJobOperatorImpl streamingJobOp;
 
+    @Value("${internal.event.listeners}")
```

Comment:
Probably it should go to separate `@Configuration`, parametrized by `@Value` and providing `@Bean List<GriffinHook>` or some `@Bean GriffinHookExecutor`. All integration points (JobService, MeasureService) would have this bean autowired.

Reply:
good point, let me change by your suggestion